Python/symtable.c

```diff
@@ -474,6 +498,21 @@ error_at_directive(PySTEntryObject *ste, PyObject *name)
       global: set of all symbol names explicitly declared as global
 */
 
+static long
+flags_in_symbols(PyObject *symbols, PyObject *name)
+{
+    if (symbols == NULL) {
+        return 0;
+    }
+    PyObject *v = PyDict_GetItemWithError(symbols, name);
+    if (v == NULL) {
+        assert(!PyErr_Occurred());
+        return 0;
+    }
+    assert(PyLong_CheckExact(v));
+    return PyLong_AS_LONG(v);
```

Comment:
This is a duplication of the existing `_PyST_GetSymbol` function.

Not sure if it's worth keeping `flags_in_symbols` just for the NULL check, or you could just do that at the call site if needed.

Reply:
Good point, I'll get rid of the function.